AnnService/inc/SSDServing/VectorSearch/SearchSsdIndex.h

```diff
@@ -73,7 +75,7 @@ namespace SPTAG {
                             }
                             else if (vectorSet != nullptr) {
                                 float dist = results[i].GetResult(j)->Dist;
-                                float truthDist = index->ComputeDistance(querySet->GetVector(i), vectorSet->GetVector(id));
```

Comment:
Why index->ComputeDistance cannot return correct distance?

Reply:
If it cannot, could you try to use SPTAG::COMMON::DistanceUtils::ComputeDistance<T>(const T* p1, const T* p2, DimensionType length, SPTAG::DistCalcMethod distCalcMethod) interface?